app/scripts/controllers/preferences.js

```diff
@@ -685,6 +686,17 @@ export default class PreferencesController {
     return this.store.getState().useLedgerLive;
   }
 
+  /**
+   * A setter for the user preference to dismiss the seed phrase backup reminder
+   * @param {bool} dismissBackupReminder- User preference for dismissing the back up reminder
+   * @returns {void}
+   */
+   async setDismissSeedBackUpReminder(dismissSeedBackUpReminder) {
```

Comment:
per the linter this is just one space over-indented: https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/9157/workflows/0ea649f7-a905-47d5-9553-3cd5a39bd86c/jobs/272007